EXRPdfReader/Views/PdfSinglePageView.m

```diff
@@ -0,0 +1,51 @@
+//
+//  PdfSinglePageView.m
+//  EXRPdfReader
+//
+//  Created by Arnold Joseph Caesar Esteban on 1/13/21.
+//  Copyright © 2021 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "PdfSinglePageView.h"
+
+@implementation PdfSinglePageView
+
+- (id)initWithFrame:(CGRect)frame{
+    if (self = [super initWithFrame:frame]) {
+        self.layer.borderColor = [UIColor lightGrayColor].CGColor;
```

Comment:
ivar

Reply:
Hmm it returns `Use of undeclared identifier '_layer'`